### DISCUSSION TITLE
内置 window.resize 事件，并触发 changeSize

### DISCUSSION THREAD

**Discussion by hustcc** [antvis/G6#5075] [Ideas]
- 参考 CR：https://github.com/antvis/G6/pull/5056

应该所有的 demo 都需要额外增加一段监听 resize 事件的代码片段。

**Comment by Aarebecca** [comment]
目前在 dumi 全局脚本中加了相关逻辑，当 graph 挂载到 window 上时会自动触发 changesize。后续会考虑添加 autoFit 属性。

https://github.com/antvis/G6/blob/4d5c037005b6f9fd4431bfff1c802c181257503c/packages/site/.dumi/app.ts#L1-L9
